cluster/vagrant-kubernetes/setup_common.sh

```diff
@@ -93,3 +94,7 @@ systemctl enable kubelet && systemctl start kubelet
 modprobe bridge
 sysctl -w net.bridge.bridge-nf-call-iptables=1
 sysctl -w net.bridge.bridge-nf-call-ip6tables=1
+
+# Storage
+yum install centos-release-gluster
+yum install --nogpgcheck -y glusterfs-fuse
```

Comment:
You should not need the `--nogpgcheck`. Packages are signed with the GPG-KEY that comes with the centos-release-gluster package.

Reply:
Why do we need glusterfs-fuse on the machine? Can't we bring that with the container?

Reply:
Is that used by the kubelet or by a flexvolume?

Reply:
kubelet. Kubernetes has a native GlusterFS mount plugin that uses glusterfs-fuse.

Reply:
ok, then it makes sense to install  it. :+1: